app/[locale]/dashboard/patient/[patientId]/page.tsx

```diff
@@ -0,0 +1,117 @@
+"use client";
+
+import {
+	Anchor,
+	Box,
+	Breadcrumbs,
+	Button,
+	Group,
+	Tabs,
+	Title,
+} from "@mantine/core";
+import Image from "next/image";
+import useAvatar from "@/hooks/use-avatar";
+import {
+	IconDots,
+	IconFileDescription,
+	IconFiles,
+	IconHeartbeat,
+	IconStethoscope,
+} from "@tabler/icons-react";
+import PatientDetails from "./_components/patient-details";
+import PatientVitals from "./_components/patient-vitals";
+import PatientBiometrics from "./_components/patient-biometrics";
+import PatientDocuments from "./_components/patient-documents";
+import VisitDetails from "./_components/visit-details";
+
+export default function PatientPage({
+	params: { patientId },
+}: { params: { patientId: string } }) {
+	const avatar = useAvatar(undefined, { seed: patientId, size: 100 });
+	return (
+		<>
+			<Breadcrumbs>
+				<Anchor href={"/dashboard"}>Dashboard</Anchor>
+				<Anchor href={"/dashboard/patients"}>Patients</Anchor>
+				<span>Patients {patientId}</span>
+			</Breadcrumbs>
```

Comment:
why didn't you use the next-intl `Link` tag?

Reply:
Anchor will work here like `Link` but will provide the design and UI of the link which adhere to Mantine design system.